test/cpp/fluid/mkldnn/CMakeLists.txt

```diff
@@ -10,6 +10,20 @@ cc_test(
        device_context
        enforce
        generated_static_op)
+       
+cc_test(
+  test_analyzer_cpu_quantize_pass
```

Comment:
To follow the style of existing cases, do we need to use same name as the source file?

Reply:
no, the generate file name can be defined other names.

Reply:
suggest to use same name just like other cases